rna/rna-quantification/rna-quantification.terra.outputs.json

```diff
@@ -0,0 +1,10 @@
+{
+  "RNAQuantification.logs": "${this.salmon_logs}",
```

Comment:
There's no obvious structure here, so could you sort these based on the name of the variable?